### DISCUSSION TITLE
Can we rely on gateway interface?

### DISCUSSION THREAD

**Discussion by lpoli** [minio/minio#14996] [Q&A] [answered]
There is gateway api for interacting with aws s3 and nas. Previously there was also GCS support.
I tried to use s3 storage with minio but dashboard shows no statistics. There are more than 10 buckets in s3 but it shows 0.

For the practical purpose, I wanted to write gateway to different storage than available in minio. But without so much features not working I am being reluctant.
Any idea?

**Comment by harshavardhana** [comment, marked as answer]
No you cannot @lpoli - gateway cannot fully function like a server - so it will never be a replacement for a server deployment features.
